backend/homework/1/app/src/main/java/org/javasession1/Main.java

```diff
@@ -0,0 +1,144 @@
+package org.javasession1;
+
+import java.util.Scanner;
+
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Main {
+    public static void main(String[] args) {
+        StudentRepository school = new StudentRepository();
+        Logging logger = new Logging();
+
+        Scanner scanner = new Scanner(System.in);
+        boolean flag = true;
+        int choice;
+        int id,age;
```

Comment:
These same variable could have defined in above line as well with choice. And do initialize with default values